src/es/home.html

```diff
@@ -0,0 +1,121 @@
+---
+title: 'Open Web Advocacy'
+metaDesc: 'Un grupo de ingenieros de software de todo el mundo que se han unido para defender el futuro de la web abierta.'
+permalink: '/es/'
+layout: 'layouts/home.njk'
+isHome: true
+---
+
+<p class="home-intro">Somos un grupo de ingenieros de software de todo el mundo que nos hemos unido para defender el
+  <strong>futuro de la web abierta </strong> proporcionando a los reguladores, legisladores y formuladores de políticas los complejos detalles técnicos que necesitan para comprender los principales problemas anticompetitivos en nuestro industria y cómo solucionarlos.
+</p>
+
+<div class="highlight">
+  <h2>¿qué está en riesgo?</h2>
+
+  <div class="group">
+    <p>El futuro entero del desarrollo de aplicaciones está en juego. Sin cambios regulatorios o
+legislativos, corremos el riesgo de perder un sistema de distribución e implementación de aplicaciones universal, libre y abierto, de escritura única e implementación en cualquier lugar, que reducirá drásticamente los costos
+para las empresas y los consumidores.</p>
+    <p>Sin estos cambios, la financiación se trasladará a ecosistemas propietarios y los guardianes
+podrán cobrar fuertes impuestos. Esto afianzará su control y reducirá la innovación en aplicaciones móviles durante muchos años.</p>
```

Comment:
I don't know how to translate taxes properly in this context. As far as I can tell, in Spanish taxes can be translated as "Tasas" o "Impuestos" but in both cases they seem to be collected by the government, not private entities.   
Maybe use just "price increase"?

Reply:
Is there a change to finalise here?

Reply:
I don't have any better suggestion, so we can just leave it as is.